src/cluster.c

```diff
@@ -548,7 +550,12 @@ void clusterInit(void) {
     memset(server.cluster->slots_keys_count,0,
            sizeof(server.cluster->slots_keys_count));
 
-    /* Set myself->port/cport/pport to my listening ports, we'll just need to
+    /* The slots -> channels map is a radix tree. Initialize it here. */
+    server.cluster->slots_to_channels = raxNew();
+    memset(server.cluster->slots_channels_count,0,
+           sizeof(server.cluster->slots_channels_count));
```

Comment:
Do we need to maintain this separate structure for the count? It seems only to ever be used as an optimization in one place, it seems easier to just loop over the rax in that instance.

Reply:
Yeah, seems like a unnecessary optimization. Just traversing the rax suffices.